localnet/kubernetes/values-ibc-relayer-cli.yaml

```diff
@@ -0,0 +1,13 @@
+prefix: "cli"
```

Comment:
Are we following standard patterns from [1]?

If so - can you link to the root source.

[1] https://hermes.informal.systems/